waffle-doppelganger/package.json

```diff
@@ -0,0 +1,57 @@
+{
+  "name": "@ethereum-waffle/doppelganger",
+  "description": "Mock smart contracts in a smart way.",
+  "version": "0.0.1",
+  "author": "Marek Kirejczyk <account@ethworks.io> (http://ethworks.io)",
```

Comment:
Let's make it Ethworks

Reply:
I'll add an issue for that and homepage change